core/src/main/java/io/grpc/transport/AbstractClientStream.java

```diff
@@ -125,8 +130,8 @@ protected void inboundHeadersReceived(Metadata.Headers headers) {
    *
    * @param frame the received data frame. Its ownership is transferred to this method.
    */
-  protected void inboundDataReceived(ReadableBuffer frame) {
-    Preconditions.checkNotNull(frame, "frame");
```

Comment:
@ejona86 IDE configuration issues are a slightly different matter and those sorts of things are violations of coding style and can (or at least should) be picked up by checkstyle. This topic, however, is a bit more squishy than that and is more in the realm of developer preference.

Yes, you could write a PR to change the style across the code base to your preference WRT `checkNotNull`, but why would you do that? It doesn't seem like that's quite the sort of thing that is being done here.  It's more of a "hey I think a few things in this file should change and while I'm in here I'll add a static import".  I see no malice in doing that.  And I see no problem with the use of a static import in this case as it does make the line more readable.